src/sql/sqlops.proposed.d.ts

```diff
@@ -119,6 +119,20 @@ declare module 'sqlops' {
 		 * @param {*} [itemLayout] Optional layout for this child item
 		 */
 		addFormItem(formComponent: FormComponent | FormComponentGroup, itemLayout?: FormItemLayout): void;
+
+		/**
+		 * Inserts a from component in a given position in the form
+		 * @param formComponent Form component
+		 * @param index index to insert the component to
```

Comment:
will add comment, right now if index is invalid, the component gets added to the end of the list. I'm not sure this is the right way to do it. maybe I change it to throw error instead